modules/zip/zip_packer.cpp

```diff
@@ -55,6 +55,15 @@ Error ZIPPacker::close() {
 	return err;
 }
 
+void ZIPPacker::set_compression_level(int p_compression_level) {
+	ERR_FAIL_COND_MSG(Z_DEFAULT_COMPRESSION > p_compression_level || p_compression_level > Z_BEST_COMPRESSION, "Invalid compression level.");
```

Comment:
While I understand the indent behind the [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions) here, we generally avoid this form in Godot's codebase, so this is preferable:

```suggestion
	ERR_FAIL_COND_MSG(p_compression_level < Z_DEFAULT_COMPRESSION || p_compression_level > Z_BEST_COMPRESSION, "Invalid compression level.");
```